src/main/java/brave/opentracing/BraveActiveSpan.java

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright 2016-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.opentracing;
+
+import brave.Span;
+import brave.Tracer.SpanInScope;
+import io.opentracing.ActiveSpan;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+
+import java.util.Map;
+import java.util.Objects;
```

Comment:
avoid adding java 8 library deps. if implementing eq/hashCode is needed, you can look at how some other brave types do that (usually it is via auto-value)